src/commands/app/db/collection/deleteOne.js

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { DBBaseCommand } from '../../../../DBBaseCommand.js'
+import { Args } from '@oclif/core'
+import chalk from 'chalk'
+
+export class DeleteOne extends DBBaseCommand {
+  async run () {
+    const { collection, filter } = this.args
+
+    try {
+      // Parse filter JSON
+      let filterObj
+      try {
+        filterObj = JSON.parse(filter)
+      } catch (error) {
+        this.error(`Invalid filter JSON: ${error.message}`)
+      }
+
+      this.log(chalk.blue(`Deleting document from collection '${collection}'...`))
+
+      const client = await this.db.connect()
+      const coll = client.collection(collection)
+
+      // Delete the document
+      const result = await coll.deleteOne(filterObj)
+
+      this.debugLogger?.info?.('Document deleted successfully:', result)
+
+      const response = {
+        collection,
+        filter: filterObj,
+        deletedCount: result.deletedCount,
+        acknowledged: result.acknowledged,
+        namespace: this.rtNamespace,
+        timestamp: new Date().toISOString(),
+        result
+      }
```

Comment:
deletedCount and acknowledged are already included in result.

```suggestion
      const response = {
        collection,
        filter: filterObj,
        namespace: this.rtNamespace,
        timestamp: new Date().toISOString(),
        result
      }
```